Sources/NIO/Bootstrap.swift

```diff
@@ -967,3 +979,44 @@ public final class NIOPipeBootstrap {
         }
     }
 }
+
+/// An channel option which can be applied to bootstrap using shorthand notation.
+/// - See: ServerBootstrap.serverOptions(_ options: [ShorthandBootstrapOption])
+public struct ShorthandBootstrapOption {
```

Comment:
Channel options have a separate file (`ChannelOption.swift`) where this might be a better fit.

Reply:
We decided to put the types in the associated bootstraps.